.github/workflows/deploy-alpha.yml

```diff
@@ -0,0 +1,33 @@
+name: deploy-alpha
+
+concurrency: alpha
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  deployment:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    environment: alpha
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Setup Node
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install Dependencies
+      - run: npm ci
+
+      - name: Bump Versions
```

Comment:
npm login is not needed before ?

Reply:
I don't think it's needed for bumping versions, the token will be needed for pushing

Reply:
Added a token, I've read around a few places and the ci should be able to push updates to github too :)